recipes/cabextract/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "cabextract" %}
+{% set version = "1.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://www.cabextract.org.uk/{{ name }}-{{ version }}.tar.gz
+  sha256: afc253673c8ef316b4d5c29cc4aa8445844bee14afffbe092ee9469405851ca7
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - conda inspect linkages {{ name }}  # [not win]
```

Comment:
There is no need to run inspect any more. That is done by default in all build by conda-build. Can you add a CLI test for the `cabextract`?